assets/dev/scss/frontend/widgets/icon-box.scss

```diff
@@ -1,48 +1,90 @@
 //
 // Icon Box
 //
-.elementor-widget-icon-box {
 
-	@media (min-width: $screen-mobile-next) {
+@mixin icon-position($device) {
+	&.elementor#{$device}-position- {
 
-		&.elementor-position- {
+		&left,
+		&right {
 
-			&left,
-			&right {
+			.elementor-icon-box-wrapper {
+				display: flex;
+			}
 
-				.elementor-icon-box-wrapper {
-					display: flex;
-				}
+			.elementor-icon-box-icon {
+				display: inline-flex;
+				flex: 0 0 auto; // Hack for Safari
+			}
 
-				.elementor-icon-box-icon {
-					display: inline-flex;
-					flex: 0 0 auto; // Hack for Safari
-				}
+		}
+
+		&right {
+
+			.elementor-icon-box-wrapper {
+				text-align: $end;
+				flex-direction: getValueByDirection(row-reverse, row);
 			}
 
-			&right {
+			.elementor-icon-box-icon {
+				margin-left: var(--icon-box-icon-margin);
+				margin-right: 0;
```

Comment:
Please test this right/left thing in RTL mode to make sure it works as it should in both LTR and RTL

Reply:
I am studying this aspect now

Reply:
@udidol 
Reverted to the original non rtl/ltr friendly version.